src/data/maksdato/useMaksdatoQuery.ts

```diff
@@ -4,8 +4,17 @@ import { get } from "@/api/axios";
 import { useQuery } from "@tanstack/react-query";
 import { minutesToMillis } from "@/utils/timeUtils";
 
-interface Maksdato {
-  maxDate: Date | null;
+interface MaksdatoDTO {
+  maxDate: MaksDato | null;
+}
+
+interface MaksDato {
+  id: string;
+  fnr: string;
+  forelopig_beregnet_slutt: Date;
+  utbetalt_tom: Date;
+  gjenstaende_sykedager: string;
+  opprettet: Date;
```

Comment:
`MaksdatoDTO` og `MaksDato`, noen som har forslag til bedre navn for `MaksDato`-objektet? Og er det `maksDato` eller `maksdato` 🤔🤔

Reply:
Det må vel være `maksdato` (i ett ord)? 🤔 Jeg kommer ikke på noe lurt samlebegrep for alle disse tingene 🤷

Reply:
Bra med rename til DTO 👍 Kommer ikke på noe bedre selv. interfacene/typene bør vel uansett starte med uppercase og feltene må vel hete det de heter pga APIet.